html/elements/input/month.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-month": {
+          "__compat": {
```

Comment:
As for the others:

               "description": "<code>type=\"month\"</code>"